contents/data/sql/SQL.md

```diff
@@ -0,0 +1,241 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+**Structured Query Language** (**SQL**) is a programming language that is specifically designed for managing (storing, retrieving and manipulating) data in a <tooltip content="relational databases store data in tables">[relational database]({{baseUrl}}/contents/data/databases/databases.html#database-models)</tooltip>. Most <tooltip content="Relational DataBase Management System">RDBMS</tooltip> distributions like MySQL, Oracle, SQL Server and PostgreSQL use the SQL language.
+However, the syntaxes used in these distributions vary slightly. These differences may be in terms of whether the syntax is case-sensitive, what  functions are available out-of-the-box, the format for specifying date and time, and so on.  
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
+As the size of your application grows, you need to look into dedicated database software packages (like SQL distributions) to store your data. It will not be enough to simply rely on in-memory solutions or a simple text file to store the data.
```

Comment:
Are you sure _SQL Distributions_ is the correct term? It didn't feel right because SQL is a language but unlike other languages, it doesn't come as a standalone installation. Rather; it comes with a database software. Besides, the phrase has a very low count in Google search results.

Reply:
Good point, I'll change it to `SQL implementations` instead

Reply:
> Earlier you used RDBMS Distributions and here you use SQL distributions. Do they refer to the same thing?

Yes, SQL is specifically for relational databases (RDBMSs). Hence SQL ~distribution~ implementation => RDBMS ?

In that sentence, I wanted to point out that SQL is the dominant language across RDBMSs.

Reply:
_RDBMS software_ seem more appropriate, as the software is sold as a database, and the SQL support is just a necessary feature?

Reply:
I considered that too. But the thing is that though most RDBMS use SQL, not all of them do so. Some exceptions are [Rel](https://reldb.org/c/) and [Dataphor](http://www.dataphor.org/About%20Dataphor.ashx). In this case, do you think that we can still use RDBMS to be synonymous with SQL?

Reply:
Perhaps you can use _SQL Software_ (or _SQL Implementations_ or _SQL Distributions_) but explain the exact meaning the first time you use it.

Reply:
Updated to this:

Unlike other languages, SQL doesn't come as a standalone installation. Rather, _SQL Software_ (or _SQL Implementations_) like MySQL, Oracle, SQL Server and PostgreSQL come with a RDBMS that uses the SQL language.